src/esp/sim/Simulator.cpp

```diff
@@ -419,28 +430,50 @@ bool Simulator::createSceneInstance(const std::string& activeSceneName) {
            curSceneInstanceAttributes->getTranslationOrigin()) ==
        metadata::managers::SceneInstanceTranslationOrigin::AssetLocal);
 
+  std::string errMsgTmplt =
+      "Simulator::createSceneInstance : Error instancing scene : " +
+      activeSceneName + " : ";
   // Iterate through instances, create object and implement initial
   // transformation.
   for (const auto& objInst : objectInstances) {
     const std::string objAttrFullHandle =
         metadataMediator_->getObjAttrFullHandle(objInst->getHandle());
     if (objAttrFullHandle == "") {
-      LOG(WARNING) << "Simulator::createSceneInstance : Unable to find object "
-                      "attributes whose handle contains "
-                   << objInst->getHandle()
-                   << " as specified in object instance attributes, so unable "
-                      "to instance object; skipping. ";
-      continue;
+      LOG(ERROR) << errMsgTmplt
+                 << "Unable to find objectAttributes whose handle contains "
+                 << objInst->getHandle()
+                 << " as specified in object instance attributes.";
+      return false;
```

Comment:
This and other warnings in this loop were changed to errors so that scene instancing would stop and inform the user if there were any issues processing the scene object instance configs specified.  This would indicate errors in the JSON that would result in objects being poorly formed, or otherwise not implemented. The thinking is that it would be better to stop with an error during training than just not have an important object in the scene.